integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledGetPowerQualityProfile.feature

```diff
@@ -1,27 +1,22 @@
 @SmartMetering @Platform
-Feature: SmartMetering Bundle - GetProfileGenericData
-  As a grid operator
-  I want to be able to retrieve profile generic data from a meter via a bundle request
+Feature: SmartMetering Bundle - GetPowerQualityProfile
+  As a grid operator
+  I want to be able to retrieve power quality profile data from a meter via a bundle request
 
   Background: 
     Given a dlms device
       | DeviceIdentification | TEST1024000000001 |
       | DeviceType           | SMART_METER_E     |
 
-  Scenario: Retrieve profile generic data as part of a bundled request
+  Scenario: Retrieve power quality profile data as part of a bundled request
```

Comment:
Je test moet volgens mij dan beginnen als volgt:
Given a dlms device
  | DeviceIdentification      | TEST1027000000001 |
  | DeviceType                | SMART_METER_E     |
  | Protocol                  | SMR               |
  | ProtocolVersion           | 5.1               |
  | Port                      | 1027              |
Ik zag dat je een bestaande test hebt aangepast. Ik ken niet alle details maar ik vermoed dat we voor zowel DSMR4.2 (TEST1024000000001) en SMR5 (TEST1027000000001) een test willen om de functionaliteit voor beide protocollen te waarborgen.